### PR TITLE
Style Function Not Work In WFS

### PR DESCRIPTION
**Pull request by mrzh4s**
Fix WFS style function not call each feature style and some updating version of WFS & WMS.

### CODE REVIEW

File: src/L.Geoserver.js [hunk 1]
```diff
@@ -82,8 +82,9 @@ L.Geoserver = L.FeatureGroup.extend({
         if (typeof that.options.style === "function") {
           for (i = 0; i < layers.length; i++) {
             that.addLayer(layers[i]);
-            if (i.setStyle) {
-              i.setStyle(that.options.style(i.feature));
+            if (layers[i].setStyle) {
```
Comment (iamtekson) [inline src/L.Geoserver.js:85]: Is this working correctly?

**Review by iamtekson — COMMENTED** [review]
Thank you very much @mrzh4s  for your PR. I have added few comments. Please kindly respond.

File: package-lock.json [hunk 2]
```diff
@@ -1,8 +1,31 @@
 {
   "name": "leaflet-geoserver-request",
-  "version": "1.0.3",
-  "lockfileVersion": 1,
+  "version": "1.2.2",
+  "lockfileVersion": 2,
   "requires": true,
+  "packages": {
```
[the remark this answers is not in the document]
Comment (mrzh4s) [reply package-lock.json:6]: surely.

File: src/L.Geoserver.js [hunk 3]
```diff
@@ -48,7 +48,7 @@ L.Geoserver = L.FeatureGroup.extend({
 
       data: {
         service: "WFS",
-        version: "1.1.0",
+        version: "2.0.0",
```
[the remark this answers is not in the document]
Comment (mrzh4s) [reply src/L.Geoserver.js:51]: yup already tested. work well. maybe i create parameter for user select version.

File: src/L.Geoserver.js (hunk 1 above)
Comment (mrzh4s) [reply src/L.Geoserver.js:85]: so far it works on my site. I use this kind of script.
```
style: function (f) {
        switch (f.properties.color) {
            case 'blue':
                console.log('blue case');
                return {
                    color: "#057af6"
                };
            case 'orange':
                console.log('orange case');
                return {
                    color: "#ec7800"
                };
            case 'red':
                console.log('red case');
                return {
                    color: "#f31869"
                };
            case 'yellow':
                console.log('yellow case');
                return {
                    color: "#f2dd08"
                };
            case 'green':
                console.log('green case');
                return {
                    color: "#00db3e"
                };
            default:
                console.log('default case');
                return {
                    color: "#717071"
                };
        }
    },
```

**Comment by mrzh4s** [comment]
updating some code and do your request.

**Comment by mrzh4s** [comment]
image

prove script work correctly for calling style in wfs and version selection

**Comment by mrzh4s** [comment]
some suggestion. to change CQL_FILTER to friendly name like filter. more approachable. 😊

**Review by iamtekson — APPROVED** [review]
Thank you!

**Comment by iamtekson** [comment]
> suggestion for filtering. name from CQL_FILTER to filter for more approachable script. 😊

In geoserver filter, we use `CQL_FILTER` as a parameter name. So I think we can leave it as it is.

**Pull request merged**